custom_components/alfen_wallbox/alfen.py

```diff
@@ -78,71 +74,68 @@ def __init__(self,
         self.properties = []
         self.licenses = []
         self._session.verify = False
-        self.keepLogout = False
-        self.wait = False
-        self.updating = False
+        self.keep_logout = False
         self.number_socket = 1
-        self._hass = hass
         self.max_allowed_phases = 1
         self.latest_tag = None
         self.transaction_offset = 0
         self.transaction_counter = 0
-        self.initilize = False
+        self.get_transactions = get_transactions
 
-        # set next update time as current time
-        self.next_update = datetime.datetime.now()
-        disable_warnings()
+        # disable_warnings()
 
-        # Default ciphers needed as of python 3.10
-        context = ssl.create_default_context()
-        context.set_ciphers("DEFAULT")
-        context.check_hostname = False
-        context.verify_mode = ssl.CERT_NONE
-        self.ssl = context
+        # # Default ciphers needed as of python 3.10
```

Comment:
Why do we need to set SSL here? Seems like Home Assistant always sets the default